### ISSUE TITLE
automatic upload when saving

### ISSUE THREAD

**Issue by Markus872** [lukasz-wronski/vscode-ftp-sync#223]
does someone work with ftp-sync addon for vs code? i have set it up and can do remote to local and local to remote sync      but i want saving localy automatically be done without telling him to do a local to remote sync. does someone know how to do that

**Comment by soyraro** [comment]
Same here

**Comment by Naphier** [comment]
Anyone ever figure this out, would make this extension suit my needs if this worked.

**Comment by yhmtsai** [comment]
After setting `"uploadOnSave": true`, I restart the VScode and it works for me

**Comment by chris-brehme** [comment]
Still not working on some projects. Does anyone has a fix for that? I did a reinitm resave and restart...still not working!

**Comment by jhmnieuwenhuis** [comment]
Same here.
uploadOnSave = true
but files are not uploaded.
When i select the file and then do a right-mouse : Ftp-sync: upload file/folder the file is uploaded.
But auto upload does not work...

**Comment by pdcemulator** [comment]
same here

**Comment by sinonkt** [comment]
Same here, I get started from init command
`Ftp-sync: Init`
And this `generatedFiles` part was included
``` json
   uploadOnSave: true,
   /* ...other configs */
   "generatedFiles": {
        "uploadOnSave": true,
        "extensionsToInclude": [],
        "path": ""
    }
```
even set `uploadOnSave: true`, restart vscode and run `Ftp-sync: Local to Remote`, again and again, everything still not sync on save.
but after I removed `generatedFiles` part its working.
hope this help...

**Comment by dfarley1** [comment]
@sinonkt That worked for me, thanks so much!  I added ".cpp" (it's a C++ project) to the include list and rebooted.  Now everything works, even for files that already exist.

**Comment by kfg81** [comment]
@sinonkt It's worked for me. Thank you so much!

**Comment by pleivac** [comment]
Many thanks @sinonkt, it works :)

**Comment by NajSurf** [comment]
Should be working fine after fix #264 Please comment if the issue still persists
